2021-10-10 00:28:46,415 INFO    MainThread:26898 [wandb_setup.py:_flush():69] setting env: {}
2021-10-10 00:28:46,415 INFO    MainThread:26898 [wandb_setup.py:_flush():69] setting login settings: {'api_key': '8bbaf831241b8525faf8ba0890cc99d295161186'}
2021-10-10 00:28:46,415 INFO    MainThread:26898 [wandb_setup.py:_flush():69] setting login settings: {'api_key': '8bbaf831241b8525faf8ba0890cc99d295161186'}
2021-10-10 00:28:46,415 INFO    MainThread:26898 [wandb_setup.py:_flush():69] setting login settings: {}
2021-10-10 00:28:46,415 INFO    MainThread:26898 [wandb_init.py:_log_setup():342] Logging user logs to /root/konst/ml-utils/training/ner_employment/wandb/run-20211010_002846-3qdgxu0w/logs/debug.log
2021-10-10 00:28:46,415 INFO    MainThread:26898 [wandb_init.py:_log_setup():343] Logging internal logs to /root/konst/ml-utils/training/ner_employment/wandb/run-20211010_002846-3qdgxu0w/logs/debug-internal.log
2021-10-10 00:28:46,415 INFO    MainThread:26898 [wandb_init.py:init():375] calling init triggers
2021-10-10 00:28:46,415 INFO    MainThread:26898 [wandb_init.py:init():382] wandb.init called with sweep_config: {}
config: {'data_path': '/root/konst/data/nerus_lenta/news_ner_dataset.csv', 'checkpoint_path': '/root/konst/model_checkpoints/ner_employment', 'model_path': 'ner_base_epoch_3_acc_1.1193_vs_0.1769_loss_0.1954_vs_0.1650.pth', 'batch_size': 128, 'epochs': 160, 'start_epoch': 0, 'test_size': 0.05, 'lam': 0, 'patience': 3, 'factor': 0.5, 'use_wandb': True, 'lr': 0.001, 'save_at': 1, 'adjust_lr_every': 1, 'num_workers': 4}
2021-10-10 00:28:46,415 INFO    MainThread:26898 [wandb_init.py:init():424] starting backend
2021-10-10 00:28:46,415 INFO    MainThread:26898 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-10-10 00:28:46,421 INFO    MainThread:26898 [backend.py:ensure_launched():135] starting backend process...
2021-10-10 00:28:46,425 INFO    MainThread:26898 [backend.py:ensure_launched():140] started backend process with pid: 26923
2021-10-10 00:28:46,425 INFO    MainThread:26898 [wandb_init.py:init():429] backend started and connected
2021-10-10 00:28:46,428 INFO    MainThread:26898 [wandb_init.py:init():477] updated telemetry
2021-10-10 00:28:46,428 INFO    MainThread:26898 [wandb_init.py:init():500] communicating current version
2021-10-10 00:28:46,918 INFO    MainThread:26898 [wandb_init.py:init():505] got version response upgrade_message: "wandb version 0.12.4 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-10-10 00:28:46,918 INFO    MainThread:26898 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-10-10 00:28:47,160 INFO    MainThread:26898 [wandb_init.py:init():540] starting run threads in backend
2021-10-10 00:28:48,000 INFO    MainThread:26898 [wandb_run.py:_console_start():1584] atexit reg
2021-10-10 00:28:48,001 INFO    MainThread:26898 [wandb_run.py:_redirect():1458] redirect: SettingsConsole.REDIRECT
2021-10-10 00:28:48,002 INFO    MainThread:26898 [wandb_run.py:_redirect():1463] Redirecting console.
2021-10-10 00:28:48,003 INFO    MainThread:26898 [wandb_run.py:_redirect():1519] Redirects installed.
2021-10-10 00:28:48,003 INFO    MainThread:26898 [wandb_init.py:init():565] run started, returning control to user process
2021-10-11 13:19:28,908 INFO    MainThread:26898 [wandb_run.py:_atexit_cleanup():1554] got exitcode: 255
2021-10-11 13:19:28,914 INFO    MainThread:26898 [wandb_run.py:_restore():1526] restore
2021-10-11 13:19:31,478 INFO    MainThread:26898 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1238
  total_bytes: 1238
}

2021-10-11 13:19:31,655 INFO    MainThread:26898 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1238
  total_bytes: 1238
}

2021-10-11 13:19:32,544 INFO    MainThread:26898 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1238
  total_bytes: 7610828
}

2021-10-11 13:19:32,646 INFO    MainThread:26898 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1238
  total_bytes: 7610828
}

2021-10-11 13:19:32,748 INFO    MainThread:26898 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 116430
  total_bytes: 7610828
}

2021-10-11 13:19:32,851 INFO    MainThread:26898 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7610828
  total_bytes: 7610828
}

2021-10-11 13:19:32,954 INFO    MainThread:26898 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7610828
  total_bytes: 7610828
}

2021-10-11 13:19:33,056 INFO    MainThread:26898 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7610828
  total_bytes: 7610828
}

2021-10-11 13:19:33,159 INFO    MainThread:26898 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7610828
  total_bytes: 7610828
}

2021-10-11 13:19:33,262 INFO    MainThread:26898 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7610828
  total_bytes: 7610828
}

2021-10-11 13:19:33,365 INFO    MainThread:26898 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7610828
  total_bytes: 7610828
}

2021-10-11 13:19:33,468 INFO    MainThread:26898 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7610828
  total_bytes: 7610828
}

2021-10-11 13:19:33,842 INFO    MainThread:26898 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7610828
  total_bytes: 7610828
}

2021-10-11 13:19:33,945 INFO    MainThread:26898 [wandb_run.py:_wait_for_finish():1676] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7610828
  total_bytes: 7610828
}

2021-10-11 13:19:35,127 INFO    MainThread:26898 [wandb_run.py:_show_summary():1831] rendering summary
2021-10-11 13:19:35,128 INFO    MainThread:26898 [wandb_run.py:_show_history():1869] rendering history
2021-10-11 13:19:35,129 INFO    MainThread:26898 [wandb_run.py:_show_files():1898] logging synced files
